python/user.py

```diff
@@ -0,0 +1,36 @@
+import os
+import sys
+import glob
+
+def startup(userpythonhome):
+    """
+    Run startup logic for this QGIS user
+    """
+    try:
+        # Start up can be a startup.py or a startup package
+        import startup
+    except ImportError:
+        pass 
+
+    expressionspath = os.path.join(userpythonhome, "expressions")
+    load_user_expressions([expressionspath])
+
+
+def load_user_expressions(paths):
```

Comment:
@wonder-sk thoughts on this user file we talked about.  startup is called from C++ and then it handles the rest of the loading in Python.